# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1548367140 0
#      Thu Jan 24 21:59:00 2019 +0000
# Branch stable
# Node ID e9e8aad7140217574ffb41837f8791c206ce3ae4
# Parent  dc04b1a27c1e302b616a6546a121bcbbd8d9d343
* .hgignore: Undo previous change.

diff --git a/.hgignore b/.hgignore
--- a/.hgignore
+++ b/.hgignore
@@ -30,17 +30,16 @@ syntax: regexp
 
 ## Emacs associated files
 (^|/)TAGS$
 (^|/)semantic.cache$
 
 ## Scripts associated with building Octave
 ^build-aux/config\.(guess|rpath|sub)$
 (^|/)build-aux/check-subst-vars\.sh$
-^build-aux/ar-lib$
 ^build-aux/depcomp$
 ^build-aux/install-sh$
 ^build-aux/ltmain\.sh$
 ^build-aux/mdate-sh$
 ^build-aux/missing$
 ^build-aux/move-if-change$
 (^|/)build-aux/subst-config-vals\.sh$
 (^|/)build-aux/subst-cross-config-vals\.sh$
